src/api/procedures/__tests__/launchOffering.ts

```diff
@@ -195,7 +207,7 @@ describe('launchOffering procedure', () => {
   });
 
   it('should throw an error if no valid Venue was supplied or found', async () => {
-    portfolio.getAssetBalances.resolves([{ free: new BigNumber(20) }]);
+    portfolio.getAssetBalances = jest.fn().mockResolvedValue([{ free: new BigNumber(20) }]);
```

Comment:
(nitty) Seems like this works, but is more annoying to write (assuming these vars are in scope)
```
 portfolio.getAssetBalances.mockResolvedValue([
      { free: new BigNumber(20), asset, locked, total },
    ]);
```

Assigning it a a new `jest.fn()` works, but seems like a weird way to get around a type error to me.

Maybe define an object `defaultBalance` and use spread? `mockResolvedValue([{ ...defaultBalance, free: new BigNumber(20) }])`?

Not a big deal, so I am OK if you want to leave it as is though